roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -40,8 +40,9 @@ openshift_logging_es_pvc_prefix: "{{ openshift_hosted_logging_elasticsearch_pvc_
 # config the es plugin to write kibana index based on the index mode
 openshift_logging_elasticsearch_kibana_index_mode: 'unique'
 
-openshift_logging_elasticsearch_proxy_image_prefix: "openshift/oauth-proxy"
-openshift_logging_elasticsearch_proxy_image_version: "v1.0.0"
+openshift_logging_elasticsearch_proxy_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
I don't think you can assume 'logging' images here since the oauth-proxy image is a generic image to be used across the cluster for any situation that requires oauth and proxy.  This should really be a 'global' var of some kind of which the value should be openshift/oauth-proxy or openshift/ocp-oauth-proxy.  @jupierce ?

Reply:
Image name we build is `openshift3/oauth-proxy`

Reply:
@jupierce im guessing thats for origin? for OCP is it the same image name just different location?

Reply:
To clarify: openshift3/oauth-proxy is the OCP image name. I'm not sure for origin.